Dockerfile

```diff
@@ -1,14 +1,15 @@
-FROM ubuntu:16.04 as builder
+FROM ubuntu:18.04 as builder
```

Comment:
I updated the contents of this Dockerfile to successfully build against llvm-6.0 / ubuntu 18.04.